### ISSUE TITLE
Pulses sent multiple times, one for each running Metabase instance

### ISSUE THREAD

**Issue by mirunapopa** [metabase/metabase#3367]
Sorry if it might already exist, but I couldn't find anything submitted which seems to be this bug.

We're sending daily pulses in the morning on some public slack channels, but they're always sent twice. I tried removing the pulse and recreating it, but the same issue happened. 

Browser version is probably not necessary here, neither operating system though. 

_DB_: Redshift
_Metabase version_: v0.19.1

Repeatable steps to reproduce: In my case, after creating a pulse and sending it manually, it sends only once, but as soon as it's set to send daily, the next day it would start being sent 2 times. 

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by camsaul** [comment]
@mirunapopa are you running more than one Metabase instance?

**Comment by mirunapopa** [comment]
@camsaul yes, we are, just got an answer on that. we could be running only one but we'd want to avoid that, in case one of the instances dies. do we need to kill one of them?

Thanks!

**Comment by camsaul** [comment]
@mirunapopa, unfortunately Metabase isn't really designed to work with multiple instances at this point in time. Turning on the Metabot will end up starting a process in both of your instances which explains why you are getting two replies when using it.

Lots of people are using Metabase in production with only a single instance without problems; if you're running on Elastic Beanstalk the Elastic Load Balancer should be set up to restart the instance if the health checks start consistently failing.

**Comment by mirunapopa** [comment]
okay, cool, thank you for clarifying! we're changing it on our side, hopefully it works out.

**Comment by tlrobinson** [comment]
Related to #2754 "Make Metabase horizontally scalable"

**Comment by sandeep89** [comment]
Once again Thanks for such a lovely tool :) 👍 
Adding more to this issue:
We were running metabase using aws ecs with 4 concurrent task running. All the backend scripts like sync analyse run on every task and make the usual processing really slow 
It would be a ideal world where all my backend scripts run on one instance and I can have a dedicated instance to server my UI traffic

**Comment by jornh** [comment]
Looks like this limitation got fixed via 
https://github.com/metabase/metabase/issues/2754#issuecomment-411258964

**Comment by camsaul** [comment]
Duplicate of #2754. Fix in 0.30.0
